src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/Serialization/TagHelperDescriptorJsonConverter.cs

```diff
@@ -444,7 +448,8 @@ private static void ReadBoundAttribute(JsonReader reader, TagHelperDescriptorBui
                                 if (indexerNamePrefix != null)
                                 {
                                     attribute.IsDictionary = true;
-                                    attribute.IndexerAttributeNamePrefix = indexerNamePrefix;
+                                    // TODO: Needed?
```

Comment:
Don't see why not

Reply:
This is what I get for leaving comments for myself instead of the team lol.

What I was asking here was if these strings are likely enough to duplicate that it's worth it to put them through the StringCache (which while fairly efficient does impose some level of CPU cost on each Deserialization). I ended up landing on "Yes, we should StringCache all of these", but if anyone has arguments in the other direction I'm open to hearing them.

Reply:
We have CPU benchmarks for deserializing TagHelpers if you're worried you can always look at the before/after:
https://github.com/dotnet/aspnetcore-tooling/blob/98ab3eb081237eaad7cffbd3d8870a131977de82/src/Razor/benchmarks/Microsoft.AspNetCore.Razor.Performance/TagHelperSerializationBenchmark.cs#L15-L16

Reply:
It looks like this may bloat the CPU a little bit, but given how small things are to begin with I don't think it's too bad:

|Scenario|Mean CPU time (in ms)|
|---------|--------------------|
|WithoutStringCache|4.3589|
|WithStringCache|11.5236|

Reply:
Not awful